Scripts/BuildPhases/LintAppLocalizedStringsUsage.sh

```diff
@@ -8,10 +8,14 @@ LINTER_EXEC="${LINTER_BUILD_DIR}/$(basename "${SCRIPT_SRC}" .swift)"
 
 if [ ! -x "${LINTER_EXEC}" ] || ! (shasum -c "${LINTER_EXEC}.shasum" >/dev/null 2>/dev/null); then
   echo "Pre-compiling linter script to ${LINTER_EXEC}..."
-  swiftc -sdk "$(xcrun --sdk macosx --show-sdk-path)" "${SCRIPT_SRC}" -o "${LINTER_EXEC}"
+  swiftc -O -sdk "$(xcrun --sdk macosx --show-sdk-path)" "${SCRIPT_SRC}" -o "${LINTER_EXEC}"
   shasum "${SCRIPT_SRC}" >"${LINTER_EXEC}.shasum"
   chmod +x "${LINTER_EXEC}"
   echo "Pre-compiled linter script ready"
 fi
 
-"$LINTER_EXEC" "${PROJECT_FILE_PATH:-$1}" # "${TARGET_NAME:-$2}"
+if [ -z "${PROJECT_FILE_PATH:=${1:-}}" ]; then
+  echo "Please provide the path to the xcodeproj to scan"
```

Comment:
What do you think of adding the "error:" prefix, which should make Xcode report this in the navigator?

```suggestion
  echo "error: Please provide the path to the xcodeproj to scan"
```

Reply:
Applied in [eb3cf3e](https://github.com/wordpress-mobile/WordPress-iOS/pull/18525/commits/eb3cf3effc1f1a1c075ea791422b90555e17336a)